src/components/Button/Button.scss

```diff
@@ -150,15 +150,15 @@ $stacking-order: (
 }
 
 .outline {
-  @include button-outline(color('ink', 'lighter'));
+  @include button-outline(var(--p-border));
 
   &.disabled {
-    @include button-outline-disabled(color('ink', 'lighter'));
+    @include button-outline-disabled(var(--p-border-disabled));
```

Comment:
The `$outline-color` parameter doesn't seem to be used anymore. https://github.com/Shopify/polaris-react/blob/main/src/styles/shared/_buttons.scss#L198-L205

Perhaps we can just remove passing in the argument entirely: `@include button-outline-disabled()`

Reply:
Alternatively this seems to be the only place this mixin is used - perhaps we just inline it here as all it does is `background: transparent; box-shadow: none;`